packages/components/src/Components/Main/Main.js

```diff
@@ -35,6 +35,7 @@ export class Main extends Component {
         };
     }
     render () {
+        // eslint-disable-next-line no-unused-vars
```

Comment:
You can also use this rule to permit spread operator siblings
https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings